packages/debug/src/browser/debug-schema-updater.ts

```diff
@@ -58,15 +138,17 @@ export class DebugSchemaUpdater {
                 url: launchSchemaUrl.toString()
             });
         }
+
+        return schema;
     }
 }
 
 // debug general schema
-const defaultCompound = { name: 'Compound', configurations: [] };
+export const defaultCompound = { name: 'Compound', configurations: [] };
```

Comment:
Does it mean that we support compound configurations now?

Reply:
No, it doesn't mean that. It's only about providing preference schema and validating a launch configuration.

Which UI do you mean?

Reply:
content assist in launch.json should not provide what is not implemented

Reply:
If a property is not valid then user can't retrieve it using plugin API.

Could we instead provide a diagnostics for content assist for `compounds` property key to warn user that compound configurations are not supported?

Reply:
When it should be stubbed somehow in the plugin ext for now without leaking anything to UI. Without adding half-baked code to other extensions.

Reply:
Could I do it in the following way?

At this point 
https://github.com/theia-ide/theia/blob/397f360422dae3dc590e58665fd526d04d7810c7/packages/debug/src/browser/debug-schema-updater.ts#L113
clone the schema. Then one of schemas could go to provide content assist. And the second one could be populated with configuration names and used for validation purpose.

This allows to validate preferences and will not affect UI.